src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
+   * @return {Array} - the picking color or null if 'selectedObjectIndex is invalid.
+   */
+  selectedObjectPickingColor() {
+    if (this.props.selectedObjectIndex < 0) {
+      return null;
```

Comment:
Then it feels to me like this should be a private util in draw-and-pick (which knows most about the picking module and picking colors anyway)?

As a general principle we should avoid adding non-essential functions to the public API, since people will start using them and it will be harder to update functionality in the future.

If we do keep it here, we kind of need to document it in the public docs: Layer.md